### DISCUSSION TITLE
Create a cloud pocketbase server

### DISCUSSION THREAD

**Discussion by brunowilliang** [pocketbase/pocketbase#1060] [Q&A] [answered]
I think it would be a lot easier, a pocketbase plan in the cloud 🚀

**Comment by benallfree** [comment, marked as answer]
https://pockethost.io :)

**Reply by vinceyoumans** [reply]
I do not see a price...  whats the catch?

**Reply by brunowilliang** [reply]
OMG 😍, can i use it in production?

**Reply by benallfree** [reply]
@vinceyoumans No catch. We will keep a free tier no matter what. Right now it is my personal project and you're welcome to use it with no restrictions. Some day there might be costs as with any hosting provider. This is an MIT open source project, you can run it yourself from https://github.com/benallfree/pockethost. Join us there for more discussion :)

**Reply by benallfree** [reply]
@Brunowilliang Yes! That is my goal too. The project started from a web game I'm building. I wasn't happy with Supabase or Firebase, so I started seeing what PocketBase could do. I really like PocketBase, but I don't want to be bothered with hosting it myself. I agree with the goal of sovereignty, and at the same time I think a hosted version provides major benefits for developers. So that's why I created it.
